packages/cli/test/mocks/encoded.composite.picture.post.json

```diff
@@ -1,41 +1 @@
-                        }
-                    ],
-                    "link": "bafyreicnnjytay6xpojywmt4mfqkciacnvftfy2o5jvti7r3qb3bx6zsjq"
-                },
-                "linkedBlock": "omRkYXRhpGRuYW1lZFBvc3Rmc2NoZW1hpWR0eXBlZm9iamVjdGckc2NoZW1heCxodHRwczovL2pzb24tc2NoZW1hLm9yZy9kcmFmdC8yMDIwLTEyL3NjaGVtYWhyZXF1aXJlZIFkdGV4dGpwcm9wZXJ0aWVzoWR0ZXh0o2R0eXBlZnN0cmluZ2ltYXhMZW5ndGgZE4hpbWluTGVuZ3RoAHRhZGRpdGlvbmFsUHJvcGVydGllc/RrZGVzY3JpcHRpb25sQSBQb3N0IG1vZGVsb2FjY291bnRSZWxhdGlvbmRsaXN0ZmhlYWRlcqNlbW9kZWxSzgECAXFxCwAJaG1vZGVsLXYxZnVuaXF1ZUyr4tGBMkJPoaeAeC5rY29udHJvbGxlcnOBeDhkaWQ6a2V5Ono2TWtwcGl5WFZtSm1ldU5xOFk1azFtV0pVcnQ0ak1Sd3A4ZVZNZ3Vtc0tOWld2eg=="
-            }
-        ]
-    },
-    "aliases": {},
-    "views": {
-        "account": {},
-        "root": {},
-        "models": {}
-    }
-}
\ No newline at end of file
+{"version":"1.0","models":{"kjzl6hvfrbw6c85iyd7kp3lmb35qwtx700zyyawertnudem21x9vi2jr2aoxeoa":[{"jws":{"payload":"AXESIGP_ZbNuWJnnw87635fEF5e58_ffZVJI4c_KkKV6Y__W","signatures":[{"signature":"wkOdoCXHFpU0mgTtVrYeRoAaV1aW5pR1XfyIaKJEibDxDHhBdD4R8OfPCclUsPgFa9SIwOlSfsHJunPi4UGvCw","protected":"eyJhbGciOiJFZERTQSIsImtpZCI6ImRpZDprZXk6ejZNa2ZrS2k0R1NyaVBqeTk3dENhaEdDVGlkeW9VUFE5WEpXeGVCUHpRdndxNzZBI3o2TWtma0tpNEdTcmlQank5N3RDYWhHQ1RpZHlvVVBROVhKV3hlQlB6UXZ3cTc2QSJ9"}],"link":"bafyreidd75s3g3sytht4htx236l4if4xxhz7px3fkjeodt6kscsxuy772y"},"linkedBlock":"omRkYXRhpWRuYW1lZ1BpY3R1cmVldmlld3OgZnNjaGVtYaVkdHlwZWZvYmplY3RnJHNjaGVtYXgsaHR0cHM6Ly9qc29uLXNjaGVtYS5vcmcvZHJhZnQvMjAyMC0xMi9zY2hlbWFocmVxdWlyZWSBY3VybGpwcm9wZXJ0aWVzoWN1cmyjZHR5cGVmc3RyaW5naW1heExlbmd0aBiMaW1pbkxlbmd0aAB0YWRkaXRpb25hbFByb3BlcnRpZXP0a2Rlc2NyaXB0aW9ub0EgUGljdHVyZSBtb2RlbG9hY2NvdW50UmVsYXRpb25kbGlzdGZoZWFkZXKjZW1vZGVsUs4BBAFxcQsACWhtb2RlbC12MWZ1bmlxdWVMwa6Gl9CrdC8P1fzwa2NvbnRyb2xsZXJzgXg4ZGlkOmtleTp6Nk1rZmtLaTRHU3JpUGp5OTd0Q2FoR0NUaWR5b1VQUTlYSld4ZUJQelF2d3E3NkE="}],"kjzl6hvfrbw6c7na7lrksq1miekc0u9seimwkwmo97eifrkma9xsw6cgvlch0zr":[{"jws":{"payload":"AXESIOv8rpLQ3FS4SGr_77UgOltasYEaXvsPloi9J45-eSBT","signatures":[{"signature":"Xwy3y8hru5c8jnoCh7-F3ef3si-0u1w47PdYnhqlG3nZeu-KVQ-xza_UnM_c01GUX3EGcbqg4T5ALkwFNXEUDA","protected":"eyJhbGciOiJFZERTQSIsImtpZCI6ImRpZDprZXk6ejZNa2ZrS2k0R1NyaVBqeTk3dENhaEdDVGlkeW9VUFE5WEpXeGVCUHpRdndxNzZBI3o2TWtma0tpNEdTcmlQank5N3RDYWhHQ1RpZHlvVVBROVhKV3hlQlB6UXZ3cTc2QSJ9"}],"link":"bafyreihl7sxjfug4ks4eq2x7562saos3lkyycgs67mhzncf5e6hh46jakm"},"linkedBlock":"omRkYXRhpWRuYW1lZFBvc3Rldmlld3OgZnNjaGVtYaVkdHlwZWZvYmplY3RnJHNjaGVtYXgsaHR0cHM6Ly9qc29uLXNjaGVtYS5vcmcvZHJhZnQvMjAyMC0xMi9zY2hlbWFocmVxdWlyZWSBZHRleHRqcHJvcGVydGllc6FkdGV4dKNkdHlwZWZzdHJpbmdpbWF4TGVuZ3RoGROIaW1pbkxlbmd0aAB0YWRkaXRpb25hbFByb3BlcnRpZXP0a2Rlc2NyaXB0aW9ubEEgUG9zdCBtb2RlbG9hY2NvdW50UmVsYXRpb25kbGlzdGZoZWFkZXKjZW1vZGVsUs4BBAFxcQsACWhtb2RlbC12MWZ1bmlxdWVMfNgiH2fYLPntQz5Aa2NvbnRyb2xsZXJzgXg4ZGlkOmtleTp6Nk1rZmtLaTRHU3JpUGp5OTd0Q2FoR0NUaWR5b1VQUTlYSld4ZUJQelF2d3E3NkE="}]},"aliases":{},"views":{"account":{},"root":{},"models":{}}}
```

Comment:
Is there a script to generate these mocks please?
How can we update them as we make breaking changes to the composites?